content/docs/04-clusters/03-edge/02-native/01-native-day2.md

```diff
@@ -0,0 +1,281 @@
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+
+
+# Upgrading Your Edge Clusters
+
+Palette uses immutable container-based OS images for the edge clusters, enabling efficient upgrades with minimal downtime. A new immutable OS image is built dynamically to include the upgraded artifacts, and this new image is swapped out with the existing one on the edge hosts. These immutable upgrades are based on the open-source project Kairos.
+
+You can initiate the upgrade of your clusters by updating the infrastructure cluster profile associated with the cluster as follows:
+
+<br />
+
+* Modify the infrastructure profile to upgrade the K8s pack to the desired version,
+	 Example: Version 1.22 to 1.23.
+
+
+* All the clusters built from this profile start to show an update notification.
+
+
+* Apply these notifications to your cluster to start the upgrade. You may need to resolve any conflicts in configuration between the two versions.
+
+
+# Scale Your Edge Cluster
+
+## Scale Up - Add Node(s)
+
+To add new nodes to a running cluster, you will need to prepare the new edge host device and add it to the node pool configuration of a running cluster. 
+
+* Prepare edge host(s) as described in the section above.
+
+
+* Register the edge host on the Palette Management Console using its unique edge host ID. The edge host status should change to ‘Ready’ within a couple of minutes once the host agent on the edge host pairs up with its configuration on the Palette Management Console. 
+
+
+* Edit the cluster node pool and increase the pool size of the desired node pool by adding the number of new nodes. 
+
+
+* Associate the newly added edge hosts with the node pool in the appliance fields that get added due to increased pool size. 
+
+
+* Save the configuration. 
+
+
+Edge host agent(s) on the newly added nodes will detect that they are now part of a cluster and perform the required installation steps to deploy the appropriate image and join the running cluster. 
+
+## Scale Down - Delete Node(s)
+
+TBD
+
+
+# Customizing Palette Edge Distribution
+
+For certain environments or specific hardware, there might be a need to deploy additional packages or device drivers on the system for them to function correctly. For example, certain devices might need specific versions of drivers on them to establish network connectivity. Spectro Cloud provides a way for the Palette Installer Image and Palette Edge Distribution Images (PXE, PO-K3S, PO-RKE2) to be customized.
+
+Follow the steps below to customize the Palette Distribution Image :
+
+
+1. Checkout to **this** Git Repository and Switch to  the v2 branch
+Copy the `.local.env.v2.template` file to `.local.env.v2`. Adjust the file settings as follows.
+
+```
+# Not supported yet.
+#PUSH_BUILD="false"
+# Flavor of OS+K8S
+FLAVOR="ubuntu-rke2"
+# Private registry to which the custom image is tagged and can be later pushed.
+IMAGE_REPO="gcr.io/spectro-dev-public/venkat/p6os-ubuntu-rke2"
```

Comment:
We probably should replace Venkat's user id with something like ${YOUR_PATH}

Reply:
the entire section 
"Customizing Palette Edge Distribution"
marking this as "to be discussed" as the snippets included are having a lot of Spectro images and other creds. Will work with Gautam on this section.

@jgautam @truegilb